lib/rules/forbid-component-props.js

```diff
@@ -22,48 +31,117 @@ module.exports = {
       recommended: false
     },
 
-    schema: [{
-      type: 'object',
-      properties: {
-        forbid: {
-          type: 'array',
-          items: {
-            type: 'string'
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          defaultAllowOnDOM: {
+            type: 'boolean'
+          },
+          forbid: {
```

Comment:
the schema also seems to be missing `allow`, which i think should probably share a schema with `forbid`? (ie, reuse the array schema in both)